### PR TITLE
move_repos_colls_624

### PR DESCRIPTION
**Pull request by gordonleacock**
Closes issue #624

### CODE REVIEW

**Comment by gordonleacock** [comment]
Jack, what is your concern about the mobile view? Are you concerned that it doesn't trigger sooner?

**Comment by jvine** [comment]
> Jack, what is your concern about the mobile view? Are you concerned that it doesn't trigger sooner?

Mobile view has header and links centered:
Screen Shot 2019-08-23 at 1 06 56 PM

**Comment by gordonleacock** [comment]
Okay, I'm seeing that the @media (max-width: 767px) isn't working on the home page. I'll fix that soon.

**Comment by gordonleacock** [comment]
ImprovedHomePageCentering

Because of where the h1 text breaks this doesn't look perfect. Might be better to right justify at small size but that is not what Jeniffer requested.

**Comment by ggeisler** [comment]
@gordonleacock Can't you just add `text-align: center;` to the `h1` in the mobile-specific CSS?

**Comment by gordonleacock** [comment]
Thanks Gary. Better centering at small window sizes with the latest push. Changes at app/assets/stylesheets/arclight/modules/mastheads.scss L29-39.

File: app/assets/stylesheets/arclight/modules/mastheads.scss [hunk 1]
```diff
@@ -28,6 +34,15 @@
     padding: ($spacer * 1.5) ($spacer / 2);
   }
 
+  .navbar-nav {
+    flex-direction: row;
```
[the remark this answers is not in the document]
Comment (mejackreed) [reply app/assets/stylesheets/arclight/modules/mastheads.scss]: Gotcha! Thanks I missed that context.

**Comment by mejackreed** [comment]
Thanks for making the requested changes @gordonleacock

**Pull request merged**